pkg/asset/internal/templates.go

```diff
@@ -714,6 +714,7 @@ metadata:
   labels:
     k8s-app: kube-dns
     kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
We don't include the addonmanager to my knowledge. If someone adds it, it'd try to manage this I believe.